README.md

```diff
@@ -34,8 +34,8 @@ X_train, X_test, y_train, y_test = evalml.preprocessing.split_data(X, y)
 #### Run AutoML
 ```python
 from evalml.automl import AutoMLSearch
-automl = AutoMLSearch(problem_type='binary')
-automl.search(X_train, y_train)
+automl = AutoMLSearch(X_train, y_train, problem_type='binary')
```

Comment:
nit pick, but it might be wise to make all these arguments named in our examples. that gives us more flexibility in the future when it comes to changing around positional argument and not breaking code. 

e.g in the future with unsupervised learning there might not be a `y` argument and the second positional argument could reference something else.

Reply:
Good point. We already have `y` as a named arg in component fit/transform. Will do so here too.